pkg/healthcheck/healthcheck_test.go

```diff
@@ -2302,12 +2304,12 @@ metadata:
 		hc.kubeAPI, err = k8s.NewFakeAPI(resources...)
 		hc.ControlPlaneNamespace = "test-ns"
 		if err != nil {
-			t.Fatalf("Unexpected error: %s", err)
+			t.Fatal(testutil.Errorf("Unexpected error: %s", err).WithAnn("Unexpected error"))
```

Comment:
What do you think about having `Error`, `Errorf`, `Fatal`, and `Fatalf` methods in `testutil` that take `testing.T` as a parameter?  I think there are a few advantages to this:
* less verbose at the call site: `testutil.Fatalf(t, "Unexpected error: %s", err)`
* easier to audit for tests that don't use the wrapper: no test should call `t.Errorf` etc directly
* Possible to add `testutil.AnnotatedErrorf(t, "Unexpected error", "Unexpected error: %s", err)` style methods to avoid the `with` design pattern.

Reply:
The problem with that approach is that if a test fails, it'll fail inside those functions in `testutil` and the test runner won't appropriately report where the failure happened.

Reply:
Does this help with that problem: https://golang.org/pkg/testing/#T.Helper ?

Reply:
Ah that's awesome, thanks!